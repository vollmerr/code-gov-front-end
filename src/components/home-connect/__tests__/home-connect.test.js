import React from 'react';

import HomeConnect from 'components/home-connect'

import { shallow } from 'enzyme';

describe('Testing <HomeConnect /> component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<HomeConnect />);
  });
  it('Testing if component renders', () => {
    expect(wrapper.exists()).toEqual(true);
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })
})