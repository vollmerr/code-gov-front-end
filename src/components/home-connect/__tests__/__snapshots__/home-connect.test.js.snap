// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HomeConnect /> component should render correct 1`] = `
<div
  className="block"
  id="connect"
>
  <div
    className="indented"
  >
    <hr />
    <h2>
      Connect
    </h2>
    <p>
      Connect with Code California.
    </p>
    <div
      className="links"
    >
      <span
        className="card"
      >
        <a
          href="https://www.govops.ca.gov/"
        >
          <i
            className="fa fa-envelope"
          />
          <h3>
            Subscribe
          </h3>
        </a>
      </span>
      <span
        className="card"
      >
        <a
          href="https://www.govops.ca.gov/"
        >
          <i
            className="fa fa-twitter-square"
          />
          <h3>
            Twitter
          </h3>
        </a>
      </span>
      <span
        className="card"
      >
        <a
          href="https://www.govops.ca.gov/"
        >
          <i
            className="fa fa-facebook-square"
          />
          <h3>
            Facebook
          </h3>
        </a>
      </span>
      <span
        className="card"
      >
        <a
          href="https://www.govops.ca.gov/"
        >
          <i
            className="fa fa-medium"
          />
          <h3>
            Medium
          </h3>
        </a>
      </span>
    </div>
  </div>
</div>
`;
